ui/gulpfile.esm.js

```diff
@@ -61,7 +70,13 @@ task(
       'distro:generate',
       'speedscope:copy_static_assets'
     ),
-    parallel('swagger:watch', 'speedscope:watch', 'webpack:dev')
+    parallel(
+      'swagger:watch',
+      'speedscope:watch',
+      'tsc:watch',
```

Comment:
Does this mean that tsc will be only run in dev watch mode and will not run when building the assert? Can we embed this behavior to the building pipeline as well, instead of leaving it in the CI only?

The principle is that, we should never have any logic that is CI only. All of these checks should be able to run locally. It enables a better develop experience.

Reply:
Totally agree, let me update.

Reply:
Done.